builtin/providers/packet/config.go

```diff
@@ -0,0 +1,18 @@
+package packet
+
+import (
+	"github.com/packethost/packngo"
+)
+
+const (
+	consumerToken = "aZ9GmqHTPtxevvFq9SK3Pi2yr9YCbRzduCSXF2SNem5sjB91mDq7Th3ZwTtRqMWZ"
```

Comment:
it is used by us to identify the source of a request, this is the one we issued for terraform. basically, it is not required, but if it is not present then we rate limit the requests. it is also not used for authentication, that is what the end user's auth token is used for. does that make sense?